src/main/java/net/minecraftforge/event/entity/player/CriticalHitEvent.java

```diff
@@ -0,0 +1,76 @@
+package net.minecraftforge.event.entity.player;
+
+import net.minecraftforge.fml.common.eventhandler.Event.HasResult;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+
+
+public class CriticalHitEvent extends PlayerEvent
+{
+    private final Entity livingTarget;
+    
+    private CriticalHitEvent(EntityPlayer player, Entity target)
+    {
+        super(player);
+        this.livingTarget = target;
+    }
+    
+    /**
+    * The attacked Entity by the Player
+    */
+    public Entity getTarget()
+    {
+        return livingTarget;
+    }
+    
+    /**
+     * CriticalHitEvent.Decision is fired to determinate is a player should make a critical hit.<br>
+     * This event is fired whenever a player attacks an Entity in
+     * EntityPlayer#attackTargetEntityWithCurrentItem(Entity).<br>
+     * <br>
+     * This event is not {@link Cancelable}.<br>
+     * <br>
+     * This event has a result. {@link HasResult}<br>
+     * DEFAULT: means the vanilla logic will determinate if this a critical hit. Else the vanilla logic is never used.<br>
```

Comment:
Second sentence unnecessary. Default -> vanilla logic. that should be clear enough as is